### PR TITLE
Implement deauth

### PR DESCRIPTION
**Pull request by Shadowghost**
Implements ability to deauth a user from trakt.

### CODE REVIEW

File: Trakt/Api/TraktURIs.cs [hunk 1]
```diff
@@ -88,12 +93,12 @@ public static class TraktUris
...
         /// The collected movies URI.
         /// </summary>
-        public const string CollectedMovies = BaseUrl + "/sync/collection/movies?extended=metadata";
+        public const string CollectedMovies = BaseUrl + "/sync/collection/movies";
 
         /// <summary>
         /// The collected series URI.
         /// </summary>
-        public const string CollectedShows = BaseUrl + "/sync/collection/shows?extended=metadata";
+        public const string CollectedShows = BaseUrl + "/sync/collection/shows";
```
Comment (crobibero) [inline Trakt/Api/TraktURIs.cs:101]: These changes seem unrelated to this PR?

Comment (Shadowghost) [reply Trakt/Api/TraktURIs.cs:101]: Indeed...

**Review by crobibero — APPROVED** [review]
